test/api.js

```diff
@@ -746,244 +746,257 @@ describe('/v1', function() {
       });
     });
 
-    describe('GET /clients', function() {
-      it('should require authorization', function() {
-        return Server.api.get({
-          url: '/clients'
-        }).then(function(res) {
-          assert.equal(res.statusCode, 401);
+    describe('client management api', function() {
+      it('should not be available on main server', function(){
+        return P.all([
+          Server.api.get('/clients'),
+          Server.api.post('/client'),
+          Server.api.post('/client/' + clientId),
+          Server.api.delete('/client/' + clientId)
+        ]).map(function(res) {
+          assert.equal(res.statusCode, 404);
```

Comment:
Wow, the diff really made a mess of this file!  Is this mostly just shuffling the existing tests so that they point to the appropriate server, or should I be reviewing anything here in more detail?